.github/workflows/os_java_ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Build with Maven
-        run: mvn -B clean install -Dfull
+        run: mvn -B -fae clean install -Dfull
```

Comment:
One suggestion: use `--fail-at-end` instead of the short version.

The short options are handy when typing commands on the command line but it's hard to guess what they mean without looking into the documentation.

I prefer the full-length options in the code because it makes the code less cryptic. It makes it more friendly to the outside world - people that don't work with Maven, newcomers, one time contributors etc.